applications/asset_tracker_v2/src/events/sensor_module_event.c

```diff
@@ -20,10 +20,10 @@ static char *get_evt_type_str(enum sensor_module_event_type type)
 		return "SENSOR_EVT_MOVEMENT_IMPACT_DETECTED";
 	case SENSOR_EVT_ENVIRONMENTAL_DATA_READY:
 		return "SENSOR_EVT_ENVIRONMENTAL_DATA_READY";
-	case SENSOR_EVT_FUEL_GAUGE_READY:
-		return "SENSOR_EVT_FUEL_GAUGE_READY";
```

Comment:
I guess this shouldn't be deleted or am I missing something?
Did you intend to move it like you did in `sensor_module_event_type`?

Reply:
That was the intention, fixed